packages/mcp/README.md

```diff
@@ -0,0 +1,29 @@
+[![npm version](https://badge.fury.io/js/orval.svg)](https://badge.fury.io/js/orval)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![tests](https://github.com/orval-labs/orval/actions/workflows/tests.yaml/badge.svg)](https://github.com/orval-labs/orval/actions/workflows/tests.yaml)
+
+<p align="center">
+  <img src="./logo/orval-logo-horizontal.svg?raw=true" width="500" height="160" alt="orval - Restfull Client Generator" />
+</p>
+<h1 align="center">
+  Visit <a href="https://orval.dev" target="_blank">orval.dev</a> for docs, guides, API and beer!
+</h1>
+
+### Code Generation
+
+`orval` is able to generate client with appropriate type-signatures (TypeScript) from any valid OpenAPI v3 or Swagger v2 specification, either in `yaml` or `json` formats.
+
```

Comment:
Looks like this README is cut and paste?  this should give detailed instructions about MCP, how to generate, how to start it etc?

Reply:
You're right, I forgot to edit the README.

Reply:
I added simple usage instructions to the `README`, but I think it will be difficult to understand unless I add a sample app and explain it through the guidelines.
Therefore, I will keep the `README` to a simple explanation and organize the surrounding information in an additional PR.

https://github.com/orval-labs/orval/pull/2015/commits/643916dd421dc99bff27728bedb8ffebcd28a569